src/solutions/how-tos/redpanda/iot-ingest-control.md

```diff
@@ -0,0 +1,455 @@
+---
+icon: aky-zilla-plus
+description: Setup a public to your Redpanda private cluster from anywhere on the internet.
```

Comment:
Setup a public ?? to your Redpanda...

This probably just needs to be reworded for IoT solution.